### DISCUSSION TITLE
Server down

### DISCUSSION THREAD

**Discussion by ArturoHDZG** [domcloud/domcloud#59] [Q&A] [answered]
Hello, I have a problem visiting my website, when I try to enter the browser gives me an error ERR_TUNNEL_CONNECTION_FAILED, in the check connection tab, the following error is observed:
No name server
This website does not have a name server. It maybe externally configured.
While in the Ngix error log it gives me the following:
1571 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream.

What is happening and how can I solve it?

**Comment by willnode** [comment, marked as answer]
Sorry this happens because of this issue: https://github.com/domcloud/domcloud/issues/58

**Reply by ArturoHDZG** [reply]
thank you!
